docroot/modules/humsci/hs_admin/hs_admin.module

```diff
@@ -200,3 +200,18 @@ function hs_admin_field_widget_complete_paragraphs_browser_form_alter(&$field_wi
     }
   }
 }
+
+/**
+ * Implements hook_field_widget_complete_WIDGET_TYPE_form_alter().
+ */
+function hs_admin_field_widget_complete_menu_link_default_form_alter(array &$field_widget_form, FormStateInterface $form_state, array $context) {
```

Comment:
can you use `hook_entity_field_access` instead?

Reply:
`hook_entity_field_access` works for hiding the "Expanded" field when editing the isolated menu item, but it doesn't work when editing the menu item in the node add/edit page. It seems that the hook is not invoked in that case, that's why we're using `hook_field_widget_complete_WIDGET_TYPE_form_alter()` instead

<img width="1180" alt="Screenshot 2024-04-08 at 10 54 30" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/ebc209a1-bffd-47a4-a52b-195231f7cf27">